### PR TITLE
Notification fixes

### PR DESCRIPTION
**Pull request by thediegobryan**

### CODE REVIEW

File: api/models.py [hunk 1]
```diff
@@ -14,22 +15,24 @@
...
+    clean_date = (instance.game_session.date).strftime("%a, %b, %d")
+    clean_time = (instance.game_session.time).strftime("%I:%M %p")
     if created:
         print(f"{instance.user.username} is pending for {instance.game_session}")
         if instance.user != instance.game_session.host:
             NotificationGameSession.objects.create(
                 sender=instance.user,
                 reciever=instance.game_session.host,
-                message=(f"Good news, {instance.user.first_name} would like you join your game on {instance.game_session.date} at {instance.game_session.time}. Please go to MyGames to respond."),
+                message=(f"Good news, {instance.user.first_name} would like you join your game on {clean_date} at {clean_time}. Please go to MyGames to respond."),
```
Comment (ypg00) [inline api/models.py:26]: My Games - not MyGames

Comment (ypg00) [inline api/models.py:26]: And "would like to join" rather than "would like you join your"

**Review by ypg00 — APPROVED** [review]
Couple in line comments for correction.

File: api/models.py [hunk 2]
```diff
@@ -14,22 +15,24 @@
...
                 game_session = instance.game_session,
             )
     else: 
         print("Guest has been updated")
         update_game_session_confirmed_field(instance.game_session.pk)
         if instance.status == "Accepted":
-            response = f"Yay! {instance.game_session.host.first_name} has confirmed your game on {instance.game_session.date} at {instance.game_session.time}. You can see all of your confirmed games on the MyGames page."
+            response = f"Yay! {instance.game_session.host.first_name} has confirmed your game on {clean_date} at {clean_time}. You can see all of your confirmed games on the MyGames page."
```
Comment (ypg00) [inline api/models.py:33]: My Games, not MyGames

Comment (thediegobryan) [reply api/models.py]: fixed

**Pull request merged**